mesos/README.md

```diff
@@ -0,0 +1,23 @@
+# Mesos with Marathon Framework
+
+[Marathon Framework](https://github.com/mesosphere/marathon) components for use as
+an optional scheduler in Deis, the open source PaaS.
+
+## Usage
+
+Please consult the [Mesos with Marathon framework](http://docs.deis.io/en/latest/customizing_deis/choosing-a-scheduler/#Mesos-with-Marathon-framework)
```

Comment:
Anchor links are actually treated as case-sensitive, so this link should be changed to lower-case or it will just bring you to the top of the web page:
`http://docs.deis.io/customizing_deis/choosing-a-scheduler/#mesos-with-marathon-framework`